addons/resource.language.en_gb/resources/strings.po

```diff
@@ -11003,13 +11012,12 @@ msgctxt "#20448"
 msgid "Could not download information"
 msgstr ""
 
+#: xbmc/video/VideoInfoScanner.cpp
 msgctxt "#20449"
-msgid "Unable to connect to remote server"
+msgid "Unable to connect to remote server. Would you like to continue scanning?"
```

Comment:
I thought we'd like to get rid of splitted up dialog messages with this PR? So if parts of this concatenated string are used elswhere, the other usecase needs a separate label IMO.